src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Interfaces/IReleaseFileRepository.cs

```diff
@@ -12,9 +12,9 @@ public interface IReleaseFileRepository
         public Task<Either<ActionResult, ReleaseFileReference>> CheckFileExists(Guid releaseId, Guid id,
             params ReleaseFileTypes[] allowedFileTypes);
         
-        public Task Delete(Guid releaseId, Guid fileId);
+        public Task Delete(Guid releaseId, Guid releaseFileReferenceId);
```

Comment:
I was talking with @ntsim recently and we thought about renaming `ReleaseFileReference` to `File` so that we have `File` and `ReleaseFile` which is why we renamed a lot of the variables of type `ReleaseFileReference` to just be a `file`.

On this branch I have already done a lot of renaming throughout the class, for example `dataFileReference` -> `dataFile`, `metaFileReference` -> `metaFile`, `sourceFileReference` -> `zipFile`, `releaseFileReference` -> `file`, `file` (IFormFile) -> `formFile`.

https://github.com/dfe-analytical-services/explore-education-statistics/compare/EES-1560

Reply:
Renamed back to original variable names :)